src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx

```diff
@@ -65,6 +54,17 @@ Other related docs:
 * [Login and password troubleshooting](/docs/accounts/install-new-relic/account-setup/troubleshoot-password-email-address-login-problems)
 * [Organization and account structure](/docs/accounts/accounts-billing/new-relic-one-pricing-users/account-user-structure)
 
+## Pricing plan factors [#pricing]
+
+We have two pricing models, and this can impact feature availability: 
+
+* Our newer [usage-based pricing model](/docs/new-relic-one-pricing-billing): Our newer pricing model gives users broad cross-platform access. For this pricing model, the main factors affecting access are your organization's [edition](https://newrelic.com/pricing), whether they have [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices), and a user's [permissions](#user-permissions). 
```

Comment:
I'm not sure how I feel about this sentence. It feels a little clunky! It could just be how it's reading with the links. But I think it would be better if it what was listed came first?

Reply:
sorry, could you go into more detail on that? i'm not sure what you mean